ui/src/components/namespaces/NamespaceForm.tsx

```diff
@@ -50,6 +49,8 @@ const NamespaceForm = forwardRef((props: NamespaceFormProps, ref: any) => {
   const [createNamespace] = useCreateNamespaceMutation();
   const [updateNamespace] = useUpdateNamespaceMutation();
 
+  const [isButtonEnabled, setIsButtonEnabled] = useState(false); // local state for instant button updates
```

Comment:
we don't need this state, please use `formik.dirty && formik.isValid && !formik.isSubmitting` directly or create the variable for it.